LayoutTests/fast/loader/onload-policy-ignore-for-frame-async-delegates.html

```diff
@@ -6,6 +6,7 @@
         if (window.testRunner) {
             testRunner.dumpAsText();
             testRunner.setCustomPolicyDelegate(true);
+            if (testRunner.skipPolicyDelegateNotifyDone) { testRunner.skipPolicyDelegateNotifyDone() }
```

Comment:
Why the weird style? Should be on the next line with no curly braces, just like in C++

Reply:
JS is slightly different than C++ in its handling of if statements with/without {} and I like to do this to be as unambiguous as possible.